core/output_modifier_test.go

```diff
@@ -0,0 +1,54 @@
+package core
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/rename-this/vhs/internal/ioutilx"
+	"gotest.tools/v3/assert"
+)
+
+func TestOutputModifier(t *testing.T) {
+	cases := []struct {
+		desc    string
+		outputs OutputModifiers
+		in      string
+		out     string
+	}{
+		{
+			desc: "no modifiers",
+			in:   "111",
+			out:  "111",
+		},
+		{
+			desc: "single modifiers",
+			in:   "1",
+			outputs: OutputModifiers{
+				&TestDoubleOutputModifier{},
```

Comment:
Would be nice if we had a single definition for this ( believe it's duplicated between core/coretest to avoid cyclic dependencies ). 
Unsure if it makes sense to drop the tests from `core` since it mostly defines the interface/structs and instead let `coretest` handle the actual testing?

Reply:
I would second this: if we can get rid of the manually duplicate code it should make things easier to maintain long term.

Reply:
I agree, I don't like this either, however I also want to keep realistic code coverage numbers, so if we put any actual tests into `coretest` we have to track coverage there which is a pain because most of the code in the package is used to test _other_ packages.

I think, for now at least, this is the best solution but I also think we should revisit this.